src/client/components/Checkbox/Checkbox.js

```diff
@@ -10,18 +10,21 @@ export type Props = {
   onChange?: (e: SyntheticInputEvent) => any,
   name: string,
   disabled?: boolean,
+  style?: any,
 }
 
 class Checkbox extends Component {
   constructor(props: Props) {
     super(props)
     this.state = {
       checked: this.props.checked,
+      disabled: this.props.disabled ? this.props.disabled : false,
```

Comment:
I moved it to the state so that checkboxes can be disabled in ChangesetList when more than two changesets are selected. The existing `handleChange` used `setState`, but 1) props cannot be changed this way, 2) the re-rendering of the checkbox cannot be forced this way (because there is no change of state) - at least as far as I've seen. What would you recommend instead?